package.json

```diff
@@ -1,13 +1,18 @@
 {
   "name": "specshell",
-  "version": "0.0.1",
+  "version": "0.0.1-rc.0",
   "description": "Control a program (typically a shell) over standard I/O for testing it",
-  "main": "index.js",
+  "main": "dist/shell.js",
   "scripts": {
     "build": "tsc",
     "lint": "tslint -p .",
     "test": "ava -- dist/test/**.js"
   },
+  "files": [
+    "dist/",
```

Comment:
The issue with including `dist/` is that we include `dist/test`. As long as they are not that heavy it's not that bad, but we might try [project references](https://www.typescriptlang.org/docs/handbook/project-references.html) to have a separate `src/` and `test/`.

Reply:
We do that elsewhere, too.  I'll give project references a try separately, opened #7.